### ISSUE TITLE
Create a UI-Tests for the happy path (Library 3)

### ISSUE THREAD

**Issue by lukas-monet** [amosproj/amos2023ws03-gui-frame-diff#192]
## User story
1. As a user
2. I want that the user interface is tested properly 
3. So that the software is working as intended 

## Acceptance criteria
* we have UI-Tests so that the whole Happy"-Path is tested
* the Happy Path is:
1. select two videos
2. adjust the hyperparameter in advanced settings
3. select a mask in advanced settings
4. press "compute and display difference" button
5. check the functionality of all skip buttons
6. jump to a certain position in the timeline

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
